plenum/test/replica/test_buffers_cleaning.py

```diff
@@ -0,0 +1,63 @@
+from plenum.server.replica import Replica
+
+
+class FakeNode():
```

Comment:
Why would you not want to test with a running node, these operations are not run in isolation?